traffic_ops/traffic_ops_golang/dbhelpers/db_helpers_test.go

```diff
@@ -408,3 +408,55 @@ func TestGetCDNIDFromName(t *testing.T) {
 	}
 
 }
+
+func TestGetSCInfo(t *testing.T) {
+	var testCases = []struct {
+		description   string
+		name          string
+		expectedError error
+		exists        bool
+	}{
+		//{
+		//	description:   "Success: Get valid SC",
+		//	name:          "hdd",
+		//	expectedError: nil,
+		//	exists:        true,
+		//},
+		{
+			description:   "Failure: SC not in DB",
+			name:          "disk",
+			expectedError: errors.New("error getting server capability info: sql: no rows in result set"),
+			exists:        false,
+		},
+	}
+	for _, testCase := range testCases {
+		t.Run(testCase.description, func(t *testing.T) {
+			t.Log("Starting test scenario: ", testCase.description)
```

Comment:
You shouldn't need to log this, since the results for the `t.Run` function will include the description in the name, and everything will be under that.